src/PointwiseFunctions/ScalarTensor/ScalarCharge.hpp

```diff
@@ -0,0 +1,90 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <utility>
+
+#include "ApparentHorizons/StrahlkorperGr.hpp"
```

Comment:
I think you can delete some of these includes, to reduce the dependencies of the ScalarTensor library

Reply:
Right. I reduced the includes.